src/main/java/edu/cornell/kfs/fp/document/service/CuDisbursementVoucherTaxService.java

```diff
@@ -0,0 +1,11 @@
+package edu.cornell.kfs.fp.document.service;
+
+public interface CuDisbursementVoucherTaxService {
+	
+	public boolean isForeignVendorWithTaxReviewRequired(String payeeTypeCode,
```

Comment:
since this is an interface, the public access modifier is redundant and can be removed